openwhisk/openwhisk.yaml

```diff
@@ -0,0 +1,100 @@
+# Task to define Runtime Parameters
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+    name: openwhisk
+spec:
+    inputs:
+        resources:
+            - name: runtime-git
+              type: git
+        params:
+            - name: TARGET_IMAGE_NAME
+              description: name of the image to be tagged and pushed
+            - name: TARGET_IMAGE_TAG
```

Comment:
Again this is not really used, I guess the tag is embedded in the image resource url  ${outputs.resources.runtime-image.url}